dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -305,6 +306,11 @@ def run_validation(self, job):
             error_list.write_all_row_errors(job_id)
             # Update error info for submission
             populate_submission_error_info(submission_id)
+
+            # Update last validated date
+            job.submission.last_validated = datetime.utcnow()
+            sess.commit()
```

Comment:
Possibly, depending on whether or not the methods called after perform queries that would rely on the updated information. I'll look into that